2.3/legal/imprint.md

```diff
@@ -0,0 +1,58 @@
+# Impressum
```

Comment:
The legal documents are in German?

You should include an header explaining why, if it's normal